docs/page-objects/items.rst

```diff
@@ -0,0 +1,47 @@
+.. _items:
+
+=====
+Items
+=====
+
+The ``to_item`` method of a :ref:`page object class <page-object-classes>` must
+return an item.
+
+An item is a data container object supported by the itemadapter_ library.
+Because itemadapter_ allows implementing support for arbitrary types, any kind
+of Python object can potentially work as an item.
```

Comment:
What do you think about mentioning that common examples of items which you can use are

* dict
* attrs instance
* dataclass instance?

Reply:
It may also worth it to add a simple example of an item definition.

Reply:
https://github.com/scrapinghub/web-poet/pull/152/commits/78d03f5a383d3471f619cb8593dc1ed4d5b81a38
https://github.com/scrapinghub/web-poet/pull/152/commits/e18ccc9570547a5dd7919158e63e949e35f3f58e